### PR TITLE
Filtering date fix

### PR DESCRIPTION
**Pull request by programmer4web**
fixes Neovici/cosmoz-omnitable/issues/118

### CODE REVIEW

File: cosmoz-omnitable-column-date.html [hunk 1]
```diff
@@ -52,7 +52,25 @@ <h3 style="margin: 0;">[[ title ]]</h3>
 				Cosmoz.OmnitableColumnBehavior,
 				Cosmoz.TranslatableBehavior,
 				Cosmoz.DateColumnBehavior
-			]
+			],
+
+			_fromInputString(value, property) {
+				let date = this.toDate(value);
```
Comment (nomego) [inline cosmoz-omnitable-column-date.html]: `const` ! :)

File: cosmoz-omnitable-column-date.html [hunk 2]
```diff
@@ -52,7 +52,25 @@ <h3 style="margin: 0;">[[ title ]]</h3>
...
+				if (date == null) {
+					return;
+				}
+				if (property === 'min') {
+					date.setUTCHours(0, 0, 0, 0);
```
Comment (nomego) [inline cosmoz-omnitable-column-date.html]: `date` will become a local date with `toDate()`, so to make this work as intended, we should not use the UTC-methods but the `setHours()` and similar.
Please ensure that the test from https://github.com/Neovici/cosmoz-omnitable/pull/108/files#diff-4775d6b705fe614d00ac79e1629929c4R190 is passing (locally).

File: cosmoz-omnitable-column-date.html [hunk 3]
```diff
@@ -52,7 +52,25 @@ <h3 style="margin: 0;">[[ title ]]</h3>
 				Cosmoz.OmnitableColumnBehavior,
 				Cosmoz.TranslatableBehavior,
 				Cosmoz.DateColumnBehavior
-			]
+			],
+
+			_fromInputString(value, property) {
```
Comment (nomego) [inline cosmoz-omnitable-column-date.html]: This doesn't have to be a helper method on its own at this point, maybe rather just call it `_fromHashString` ?

Comment (programmer4web) [reply cosmoz-omnitable-column-date.html]: _fromInputString is an function originally defined in range behavior.

Comment (nomego) [reply cosmoz-omnitable-column-date.html]: Ok but then we don't need to override `_fromHashString` rather since it's the same as the one inherited?

Comment (nomego) [reply cosmoz-omnitable-column-date.html]: Ah right it was a second argument, didn't see that!
Anyway gonna merge this now before I mess it up more :)

Comment (programmer4web) [reply cosmoz-omnitable-column-date.html]: I have already updated the original _fromHashString with the second argument. :)

**Pull request merged**